protocols/types/contracts/Types.sol

```diff
@@ -25,28 +25,34 @@ library Types {
 
   bytes constant internal EIP191_HEADER = "\x19\x01";
 
+  struct Rule {
```

Comment:
I think this name still collides with the Delegates.

Reply:
Yeah it's the exact same struct - it needs to be removed from Delegate and imported into Delegate from this library